parser/transcript/src/test/java/de/danoeh/antennapod/parser/transcript/PodcastIndexTranscriptParserTest.java

```diff
@@ -0,0 +1,157 @@
+package de.danoeh.antennapod.parser.transcript;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import de.danoeh.antennapod.parser.transcript.PodcastIndexTranscriptParser;
+import de.danoeh.antennapod.model.feed.Transcript;
+
+@RunWith(RobolectricTestRunner.class)
+public class PodcastIndexTranscriptParserTest {
+    private static String srtStr = "1\n"
+            + "00:00:00,000 --> 00:00:02,730\n"
+            + "John Doe: Promoting your podcast in a new\n\n"
+            + "2\n"
+            + "00:00:02,730 --> 00:00:04,600\n"
+            + "way. The latest from PogNews.\n\n"
+            + "00:00:04,730 --> 00:00:05,600\n"
+            + "way. The latest from PogNews.";
+    private static String jsonStr = "{'version': '1.0.0', "
+            + "'segments': [ "
+            + "{ 'speaker' : 'John Doe', 'startTime': 0.8, 'endTime': 1.9, 'body': 'And' },"
+            + "{ 'speaker' : 'Sally Green', 'startTime': 1.91, 'endTime': 2.8, 'body': 'this merges' },"
+            + "{ 'startTime': 2.9, 'endTime': 3.4, 'body': 'the' },"
+            + "{ 'startTime': 3.5, 'endTime': 3.6, 'body': 'person' }]}";
+
+    // segments is missing
+    private static String jsonStrBad1 = "{'version': '1.0.0', "
```

Comment:
I think if the string is only used in one single test method, it is more easy to read the test if the string is defined inside the test method. Then one can have everything on one screen without having to scroll.